helm-chart/splunk-enterprise/templates/enterprise_v4_indexercluster.yaml

```diff
@@ -21,6 +21,9 @@ items:
 {{ toYaml . | indent 6 }}
   {{- end }}
   spec:
+  {{- if $.Values.indexerCluster.serviceAccount }}
```

Comment:
Nit: any reason why this is moved up? Just re-aligning the code?

Reply:
yes